uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/main/java/org/uberfire/ext/security/management/client/widgets/management/editor/group/workflow/GroupCreationWorkflow.java

```diff
@@ -313,4 +330,14 @@ void showError(final Throwable throwable) {
         errorEvent.fire(new OnErrorEvent(GroupCreationWorkflow.this,
                                          throwable));
     }
+
+    void onCreateGroupEvent(@Observes final CreateGroupEvent event) {
```

Comment:
@caponetto I moved this event listener to this GroupCreationWorkflow.java to have better separation of concern and easy code coverage for this function. Since it was failing the sonar cloud code coverage.

Reply:
Thanks, @RishiRajAnand. It makes sense.

Reply:
Thanks